### PR TITLE
Fixed version check for pgRouting v3 support

### LINKED ISSUE #126: Database does not appear
Windows 10
PostGres 12.2
PostGIS 3.0
pgrouting 3.0.0-rc1
pgroutinglayer 3.0.0
QGIS 3.12

The database is loaded with OSM and works and display the layer properly in QGIS.
The pgroutinglayer plugin Database selection widget is blank.
The database has a username and password, however it is not on the standard postges port 5432.

Is there a configuration requirement for the connection? The connection appears correctly in the QGIS Browser.

Quickly reviewed the connectors/postgis.py, and am wondering the issue is related to this code not acquiring active database connections.

Thank you.

### PR DESCRIPTION
**Pull request by cayetanobv**
Fixes #126 .

Changes proposed in this pull request:
- fixed pgr_version check for v3
- updated docs

@pgRouting/admins

### CODE REVIEW

**Comment by agiudiceandrea** [comment]
Hi @cayetanobv, did you forget to merge these commits?

**Comment by cayetanobv** [comment]
Thanks @agiudiceandrea 
Yes, I forgot to merge because v3 was released after this change.

File: pgRoutingLayer_utils.py [hunk 1]
```diff
@@ -129,7 +129,7 @@ def getPgrVersion(con):
     ''' returns version of PostgreSQL database. '''
     try:
         cur = con.cursor()
-        cur.execute('SELECT version FROM pgr_version()')
+        cur.execute('SELECT pgr_version FROM pgr_version()')
```
[the remark this answers is not in the document]
Comment (cayetanobv) [reply pgRoutingLayer_utils.py:132]: LGTM

**Comment by cayetanobv** [comment]
@dkastl @cvvergara I think we can merge this PR. Culd you approve PR?

**Comment by agiudiceandrea** [comment]
Hi @cayetanobv,

in https://github.com/pgRouting/pgRoutingLayer/blob/master/README.md is stated that:

> Dependencies
> pgRouting v2.x up and running to use this plugin.

So, this should be changed to **pgRouting v3.x**, isn't it?

It is also stated that:

> Additionally, QGIS needs python-psycopg2 installed to be able to connect to the database.

Is **python-psycopg2** yet needed? Or is **python3-psycopg2** needed instead?

**Review by dkastl — APPROVED** [review]

**Comment by cayetanobv** [comment]
yes, you are right @agiudiceandrea . Docs should be fixed. Could you update this versions and push it with this PR so we can merge all together?

File: README.md [hunk 2]
```diff
@@ -43,6 +43,6 @@ This plugin can be installed using the QGIS Plugin Manager.
 
 ### Dependencies
 
-pgRouting v2.x up and running to use this plugin.
+pgRouting v3.x up and running to use this plugin.
 
 Additionally, QGIS needs python-psycopg2 installed to be able to connect to the database.
```
Comment (agiudiceandrea) [inline README.md:48]:
```suggestion
Additionally, QGIS needs python3-psycopg2 installed to be able to connect to the database.
```

File: docs/index.md [hunk 3]
```diff
@@ -37,7 +37,7 @@ This plugin can be installed using the QGIS Plugin Manager.
 ### Dependencies
 
 - This plugin runs only with QGIS 3.x and Python 3.
-- pgRouting v2.x up and running to use this plugin.
+- pgRouting v3.x up and running to use this plugin.
 - Additionally, QGIS needs python-psycopg2 installed to be able to connect to the database.
```
Comment (agiudiceandrea) [inline docs/index.md:41]:
```suggestion
- Additionally, QGIS needs python3-psycopg2 installed to be able to connect to the database.
```

**Comment by cayetanobv** [comment]
Thanks @agiudiceandrea

**Comment by cayetanobv** [comment]
Sorry @agiudiceandrea but I didn't notice that you are pushing to my branch and you havn't permissions so your code wasn't merged. Could you open a new branch with these changes and open a new PR? After that, I can merge your changes.
Thanks again.

**Pull request merged**